netci.groovy

```diff
@@ -274,7 +274,7 @@ Utilities.addGithubPRTrigger(prCCJob, 'Code Coverage Windows Debug', '@dotnet-bo
     def newRollingJob = job(Utilities.getFullJobName(project, jobName, false)) {
         label('windows-elevated')
         steps {
-            batchFile("build.cmd /p:Configuration=${configuration} /p:WithCategories=OuterLoop")
+            batchFile("build.cmd /p:Configuration=${os}_${configuration} /p:WithCategories=OuterLoop")
```

Comment:
The issue we are fixing seems to be isolated to the Configuration parameter. OSGroup as it is being used in any of the calls to Build.cmd is working as expected.